replicator-security/README.md

```diff
@@ -0,0 +1,128 @@
+# Replicator security configuration demos
+
+## Purpose 
+
+This repository provides docker-compose environments representing various Replicator security configurations. 
```

Comment:
```suggestion
This demo provides several docker-compose environments for Replicator configured with various security configurations.
```

Reply:
For consideration: tell users to diff the files to the non-secure one to see the delta?